cmd/thanos/receive.go

```diff
@@ -767,6 +769,8 @@ func (rc *receiveConfig) registerFlag(cmd extkingpin.FlagClause) {
 
 	cmd.Flag("receive.tenant-header", "HTTP header to determine tenant for write requests.").Default(receive.DefaultTenantHeader).StringVar(&rc.tenantHeader)
 
+	cmd.Flag("receive.tenant-certificate-attribute", "Use TLS client certificate's attribute to determine tenant for write requests. Must be one of organization, organizationalUnit or commonName.").Default("").StringVar(&rc.tenantAttribute)
+
```

Comment:
I think we could use `EnumVar` instead of `StringVar` here, since there are predetermined options possible only.

Reply:
Good point, updated the flag type.